includes/class-wc-payments.php

```diff
@@ -645,6 +646,14 @@ public static function init() {
 		add_action( 'wp_enqueue_scripts', [ __CLASS__, 'enqueue_assets_script' ] );
 
 		self::$duplicate_payment_prevention_service->init( self::$card_gateway, self::$order_service );
+
+		// phpcs:disable
+		if ( isset( $_GET['load'] ) ) {
+			$payment_service = new PaymentService();
+			var_dump( $payment_service );
+			exit;
+		}
+		// phpcs:enable
```

Comment:
> May setup a new PHPUnit test file. That would simply validate PaymentService::class to its full qualified class name.

Good idea, done now.

> Also use the namespace for it, something like these in Woo core.

I did use a namespace for the test, as was my plan for when I had to write another one. I did not see a need to add it to the autoloader though, it seems to just work, as with other PHP files during unit tests 🤔

Reply:
>  I did not see a need to add it to the autoloader though, it seems to just work, as with other PHP files during unit tests 

I think that's because PHPUnit will load all `.php` files based on [its configuration](https://github.com/Automattic/woocommerce-payments/blob/9da528c0dd6efef2c0fabce99123c97617acb1fb/phpunit.xml.dist#L11-L12). 

If we also config the autoload for these test files, suddenly these test files will be available outside of test runners too. That's unnecessary IMO.